### DISCUSSION TITLE
How to access to the objects from a transformer ?

### DISCUSSION THREAD

**Discussion by cedced19** [alan-turing-institute/sktime#2680] [Q&A]
Hi,

For research purpose I want to access to the shapelet generated by this code:
```python
rst = RandomShapeletTransform(
        n_shapelet_samples=100,
        max_shapelets=10,
        batch_size=20,
        n_jobs=3
)
transformers = [("RST 1", rst, [0]),("RST 2", rst, [1]),("RST 3", rst, [2])]
t = ColumnTransformer(transformers=transformers)
t.fit(X_train, Y_train)
```

Why is the shapelet array empty in this case ? 

![image](https://user-images.githubusercontent.com/5300939/169716170-6946e98b-c044-423c-9584-ea08b9384ec0.png)

With the following code I can access to the shapelets:
```python
t = RandomShapeletTransform(
        n_shapelet_samples=100,
        max_shapelets=10,
        batch_size=20,
        n_jobs=3
)
t.fit(X_train, y_train)
```
Therefore I think this is due to my missunderstanding of how Python objects work there. If anyone can help me, I would appreciate.

Thank you for your time.

**Comment by fkiraly** [comment]
It should be in the *fitted* attribute, `transformers_`.

That is, look in `transformers_[0][1]`, it should be there.

**Comment by cedced19** [comment]
Thank you a lot for your answer! It was that! :relaxed:
I anyone want to get the same for a `ColumnEnsembleClassifier` you should check `estimators_` for the fitted attibute obviously. :relieved:

**Comment by fkiraly** [comment]
And... if anyone finds the courage in their hearts to implement `get_fitted_params` functionality for the convenient, unified access interface, it would be much appreciated 😄 
See issue https://github.com/alan-turing-institute/sktime/issues/1497
